webplugin/plugin.js

```diff
@@ -129,6 +129,28 @@ function removeElementFromHtmlById(elementIdArray) {
         element && element.parentNode.removeChild(element);
     }
 }
+
+function appendIframeAfterBodyLoaded() {
+    if (document.body) {
+        appendIframe();
+    } else if ('readyState' in window.parent.document) {
+        appendIframeAfterLoadedBody();
+    } else {
+        window.onload = function () {
+            appendIframe();
+        };
+    }
+
+    function appendIframeAfterLoadedBody() {
```

Comment:
rename this function to avoid confusion with appendIframeAfterBodyLoaded

took me sometime to realize that these are different names

Reply:
changed the name